src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+4.4
+-----
+
+ * added a `ignore_invalid_datetimes` context option to allow invalid date strings when denormalizing without 
```

Comment:
This name could be confusing because this option doesn't really made the serializer "ignoring" the passed value, it keeps it. What do you think about changing to `datetime_allow_invalid`?

Reply:
Change was made (and tested)